File: Php/InserirFiles.php
<?php
$servername = "127.0.0.1";
$username = "root";
$password = "P@ssword12";
$dbname = "components";

$conn = new mysqli($servername, $username, $password, $dbname);

if ($conn->connect_error) {
    die("Error de conexión a la base de datos: " . $conn->connect_error);
}

$values = "";

for ($i = 1; $i <= 100000; $i++) {
    // Generar un valor aleatori entre 1 y 999 
    $valor = rand(1, 999);
    // Generar un valor aleatori entre 1 y 999 
    $random = rand(1, 999);
    //Agregar els valors generats al string $values, separant-los per comes i ficant-los entre parentesis
    $values .= "($valor, $random),";
}

// Eliminar la ultima coma del string $values 
$values = rtrim($values, ",");

$sql = "INSERT INTO filas(Valor, Random) VALUES $values";

// Executar la consulta SQL 
if ($conn->query($sql) === TRUE) {
    echo "Insertades 100,000 files en a la taula files.";
} else {
    echo "Error a la consulta: " . $conn->error;
}

// Cerrar la conexión a la base de datos
$conn->close();

File: Java/Components Java/src/InserirFiles.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;

public class InserirFiles {
    public static void main(String[] args) {
        // Establecer los parámetros de conexión a la base de datos
        String servername = "127.0.0.1";
        String username = "root";
        String password = "P@ssword12";
        String dbname = "components";

        String url = "jdbc:mysql://" + servername + ":" + "/" + dbname;

        try (Connection conn = DriverManager.getConnection(url, username, password)) {
            Statement statement = conn.createStatement();

            StringBuilder values = new StringBuilder();
            for (int i = 1; i <= 100000; i++) {
                int valor = (int) (Math.random() * 999) + 1;
                int random = (int) (Math.random() * 999) + 1;
                values.append("(").append(valor).append(", ").append(random).append("),");
            }

            values.deleteCharAt(values.length() - 1);

            String sql = "INSERT INTO filas(Valor, Random) VALUES " + values;

            statement.executeUpdate(sql);

            System.out.println("S'han insertat 100,000 files a la taula filas.");

        } catch (SQLException e) {
            System.out.println("Error a l'executar la consulta: " + e.getMessage());
        }
    }
}
